spec/requests/api/v1/auth/registrations/update_spec.rb

```diff
@@ -0,0 +1,63 @@
+describe 'PUT api/v1/auth/registrations', type: :request do
+  let(:user) { create(:user) }
+  let(:first_name) { 'Darth' }
+  let(:last_name) { 'Vader' }
+  let(:params) do
+    {
+      first_name: first_name,
+      last_name: last_name
+    }
+  end
+
+  subject do
+    put api_v1_user_registration_path, params: params, headers: headers, as: :json
+    response
+  end
+
+  include_examples 'not singed in examples'
```

Comment:
for all of these, create a context wrapping it that says `when the user is not signed in`

Reply:
let's standardize where correct contexts go first or last, but stick to one